gcloud-java-core/src/test/java/com/google/cloud/ServiceOptionsTest.java

```diff
@@ -215,6 +219,16 @@ public void testBaseEquals() {
   }
 
   @Test
+  public void testLibraryName() {
+    assertEquals(LIBRARY_NAME, OPTIONS.libraryName());
+  }
+
+  @Test
+  public void testApplicationName() {
```

Comment:
Yes but that will cause the "fake" manifest to be put into the tests jar. Unless we exclude it via maven configuration. I'll give this some thoughts but I would leave it for a future PR.